src/setStateIndicator/randomNum.js

```diff
@@ -1,16 +1,8 @@
-export const getRandomNumber = (max) => {
-  return Math.floor(Math.random() * max);
-};
-
-export const setStateDependingRandomNum = (maxNum, setIndicator) => {
+export const setStateDependingRandomNum = (setIndicator) => {
```

Comment:
I'd name this function `setIsSick`. No need to reflect the internal implementation of the function in its name. You could change the implementation not to be a random number, but you will also have to change the function name then, which is not cool.

Reply:
ok